modules/rostests/kmtests/ntos_mm/NtCreateSection_drv.c

```diff
@@ -413,17 +414,38 @@ TestIrpHandler(
     {
         Fcb = IoStack->FileObject->FsContext;
         ok(Fcb != NULL, "Null pointer!\n");
-        if (IoStack->FileObject->SectionObjectPointer != NULL &&
-            IoStack->FileObject->SectionObjectPointer->SharedCacheMap != NULL)
+
+        if (IoStack->FileObject->SectionObjectPointer == NULL)
+        {
+            ok(!CcIsFileCached(IoStack->FileObject), "File is cached\n");
```

Comment:
I don't get it.
SectionObjectPointer being set doesn't necessarily mean that the file is cached. It just means the FSD set it.
On the other hand, SectionObjectPointer being NULL means that the file is not cached.
Here you're not more explicit, just wrong.

Reply:
For my earlier stated goal, a "CANNOT fail" is perfectly fine: I am only interested in having an `ok()` to increase the stats, and I prefer a similar check than an `ok(true)`. I like logs that tell what tests are doing, rather than failing in the void.
Your mileage may vary.